### ISSUE TITLE
Feature Request: Decrease Lighthouse Progressive Web App Errors

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#103]
### Describe the feature

The site currently has a number of errors when checked when checking for the Progressive Web App standards in Lighthouse.

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110366940-ee57ef00-803e-11eb-80d2-9f0457647fab.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the Progressive Web App of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse errors for Progressive Web App is decreased
